yarn.lock

```diff
@@ -4117,6 +4117,7 @@ __metadata:
     "@glint/template": "npm:^1.4.0"
     "@hashicorp/design-system-tokens": "npm:^2.1.0"
     "@hashicorp/ember-flight-icons": "npm:^5.1.2"
+    "@hashicorp/flight-icons": "npm:^3.3.0"
```

Comment:
current latest version is `3.4.0` (hence the double entry in `yarn.lock` I think)
https://www.npmjs.com/package/@hashicorp/flight-icons?activeTab=versions